backend/bottari/src/main/java/com/bottari/service/BottariTemplateService.java

```diff
@@ -157,4 +161,16 @@ private void validateOwner(
             throw new IllegalArgumentException("본인의 보따리 템플릿이 아닙니다.");
         }
     }
+
+    private void increaseTakenCount(
+            final BottariTemplate bottariTemplate,
+            final Member member
+    ) {
+        if (!bottariTemplateHistoryRepository.existsByBottariTemplateIdAndMemberId(
+                bottariTemplate.getId(), member.getId())) {
```

Comment:
이 부분은 조건식을 메서드로 추출하고, 메서드 명으로 의도를 나타내보는 게 어떨까?


```suggestion
        if(alreadyTakenBottariTemplate()) {
           return;
        }
```

이런 형태도 가능할 것 같고!

`increaseTakenCount()`라는 메서드명이 어떤 행위를 하는 건지 잘 나타내고 있다고는 생각이 들지만,
나의 경우에는 구현 방식을 이미 들은 상태라 조건문을 읽어보는데, 어떤 의미인지 이해했지만
그러지 못할 경우도 있을 것 같아서!

가볍게 생각만 해주면 좋을 것 같아 😁

Reply:
그렇네! 나도 훌라 의견과 동일해~ 메서드 분리해볼게!